packages/orbit-components/src/FeatureIcon/FeatureIcon.stories.tsx

```diff
@@ -1,33 +1,33 @@
-import * as React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
 
 import { NAME_OPTIONS } from "./consts";
 
 import FeatureIcon from ".";
 
-export default {
+const meta: Meta<typeof FeatureIcon> = {
   title: "FeatureIcon",
-};
-
-export const Playground = ({ name, dataTest }) => {
-  return <FeatureIcon name={name} dataTest={dataTest} />;
-};
+  component: FeatureIcon,
 
-Playground.story = {
   parameters: {
     info: "Visit Orbit.Kiwi for more detailed guidelines.",
   },
-};
 
-Playground.args = {
-  name: NAME_OPTIONS.TICKETFLEXI,
-  dataTest: "test",
```

Comment:
Is there any reason why we don't keep the `dataTest` anymore?

Reply:
I got information that [we won't include](https://github.com/kiwicom/orbit/pull/4437#pullrequestreview-2220818766) `dataTest` attribute to Storybook.

Reply:
Ok, thanks for the info.